benches/mantis/deploy.ncl

```diff
@@ -1,83 +1,95 @@
+        "enode://39b925ba0beffdb80859a0ab34895c98bb61bd20d686ccd27f8c5a04dddc82b712081fd11bfd43f3bc08b00423a5ff8fee70b8a22dcc95e85537b2084dc6816a@mantis-staging-0.mantis.ws:33000?discport=33500",
+        "enode://cbd80c7f72a889101b7f23d51be2de7e3f1f46ad3b25c438e959e24e08f03bd9fe833460e84b60174d4eb120af3b127389c4606f81c842943c4922cab384a234@mantis-staging-1.mantis.ws:33001?discport=33501",
+        "enode://0e63642be49c5a092569aa01663fcda1505362cd0ac41e24ff9296ab80c97af135fb6fb247273631a3a11257774f39ed882d72a20fd45131e53e9015adf6b9e5@mantis-staging-2.mantis.ws:33002?discport=33502",
+        "enode://3ee3641a25cfc611ba54a898260af7768ecf0643f06aefedf853864ed433d5ad6265eeb24abcc4d6f6ee90a1eac6c1fbf157fc05fd8e28e194dfc864cb56058e@mantis-staging-3.mantis.ws:33003?discport=33503",
+        "enode://907842e336fc757bbfde70368aef329714aa627e72e5da687f31b097fa71a59f36404aebbc83885c9b515270042e025a6788b700c314ee8bc68099dcff32afcd@mantis-staging-4.mantis.ws:33004?discport=33504",
+      ],
+    }
+  in
+  let params =
+    {
+      namespace = vars.namespace,
+      datacenters
+        | Array (lib.contracts.OneOf [
+          "eu-central-1",
+          "us-east-2",
+          "eu-west-2"
+        ])
+        | default
+        = [ "eu-central-1", "us-east-2" ],
+      fqdn = "mantis.ws",
+      networkConfig
+        | String
+        | default
+        = m%"
                     mantis.blockchains.testnet-internal-nomad.bootstrap-nodes = [
                       %{string.join ",\n" bootstrapNodes."%{vars.namespace}"})
                     ]
                     "%,
-    logLevel | default = "INFO",
-    loggers = defaultLoggers,
-    job = vars.job,
-    role | [| `passive, `miner, `backup |] = vars.role,
-  } in
-
-  let jobDefs = {
-    Mantis = import "jobs/mantis.ncl" params,
-  } in
-
-  let revisions = {
-    mantisOpsRev = "7404801c6fcb3f12438d3c82e34ad1918bb5f0a5",
-    mantisRev = "d6cada089a10e3d9988e27d695259a216f6e21f8",
-    morphoRev = "e47b74d5e7a78bf665758927336a28a915b3e596",
-  } in
-
-  let miner_ = jobDefs.Mantis & {
-    role = `miner,
-    mantisRev = revisions.mantisRev
-  } in
-
-  let namespaces = {
-    "mantis-staging".jobs = {
-      miner = miner_,
-      passive | (passive & {count = 1})
-    },
-  } in
-
+      logLevel | default = "INFO",
+      loggers = defaultLoggers,
+      job = vars.job,
+      role | [| `passive, `miner, `backup |] = vars.role,
+    }
+  in
+  let jobDefs =
+    {
+      Mantis = import "jobs/mantis.ncl" params,
+    }
+  in
+  let revisions =
+    {
+      mantisOpsRev = "7404801c6fcb3f12438d3c82e34ad1918bb5f0a5",
+      mantisRev = "d6cada089a10e3d9988e27d695259a216f6e21f8",
+      morphoRev = "e47b74d5e7a78bf665758927336a28a915b3e596",
+    }
+  in
+  let miner_ =
+    jobDefs.Mantis
+    & {
+      role = `miner,
+      mantisRev = revisions.mantisRev
+    }
+  in
+  let namespaces =
+    {
+      "mantis-staging".jobs = {
+        miner = miner_,
+        passive | (passive & { count = 1 })
```

Comment:
It's not that important, but I wonder if there are many language where lists and/or record have the additional spacing after `{`/`[` and before `}`/`]`? It looks a tad alien to me, especially for lists/arrays, but I can't say for sure. Is there any inspiration here?